src/book.py

```diff
@@ -180,39 +217,79 @@ def tokenize(self, on, rem_stopwords=True, stopword_lang='english',
                     str.maketrans('', '', string.punctuation + '”“’'))
         return token
 
-    def snippet(self, length, on, non_cont=False, inplace=False):
+    def snippet(self, length, on, groups=1, non_cont=False, randomized=False, rem_stopwords=False, ret_as_arr=False, with_replace=True, random_seed=0, inplace=False):
```

Comment:
The default value of `randomized` should be true since this was the previous default behavior.

Reply:
Ok.